src/components/bluebutton.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+const buttonStyle = {
+  margin: '10px 10px 10px 0',
+  width: '100%',
+  backgroundColor: '#09628D',
+  color: 'white',
+  textalign: 'center',
+  height: '40px',
+  lineheight: '40px',
+};
+
+const Button = React.createClass({
+  render() {
+    return (
+      <button
+        className="btn btn-default"
```

Comment:
Here you restyled button class that is taken from http://getbootstrap.com/ framework. Plan it to remove it from project.